metrics5/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2012-2020 The Feign Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.github.openfeign</groupId>
+    <artifactId>parent</artifactId>
+    <version>10.7.5-SNAPSHOT</version>
+  </parent>
+  <artifactId>feign-metrics5</artifactId>
+  <name>Feign Metrics5</name>
+  <description>Feign Dropwizard Metrics 5</description>
```

Comment:
I'd like to suggest an alternative, [Micrometer](https://micrometer.io/docs).  It can wrap Dropwizard and will allow this to be integrated directly into Spring Boot/Cloud, since Micrometer is their preferred metrics collector.

Reply:
My plan is to add that one later, added metrics 5 first cause is the one I'm using and gets values for me faster

Reply:
They both do the same thing. If you want to have multiple implementations I’d like to suggest that we rename this to metrics-dropwizard